gui/additionsPane.py

```diff
@@ -57,6 +58,7 @@ def __init__(self, parent):
         projectedImg = BitmapLoader.getImage("projected_small", "gui")
         gangImg = BitmapLoader.getImage("fleet_fc_small", "gui")
         cargoImg = BitmapLoader.getImage("cargo_small", "gui")
+        notesImg = BitmapLoader.getImage("skill_small", "gui")
```

Comment:
I like the current skillbook icon. But yeah, if you find another icon from the EVE image export, simply skrink it to 16x16 and include it. But as it stands now, looks good